jaworski_cellseg/label_counting_widget.py

```diff
@@ -1,9 +1,50 @@
 from magicgui import magicgui
 import napari
 from skimage.measure import regionprops
```

Comment:
```suggestion
from skimage.measure import regionprops, RegionProperties
```

Reply:
It seems RegionProperties cant be imported directly from skimage.measure. I did a workaround to support the params hints